src/Component/Calculator.js

```diff
@@ -1,52 +1,72 @@
-import React from 'react';
+/* eslint-disable */
+import React, { useState } from 'react';
+/* eslint-enable */
+import calculate from '../logic/calculate';
 
```

Comment:
- [ ] Please don't diabale eslint. eslint rules are here to help you write a clean code.